frontend/app/src/people/widgetViews/summaries/WantedSummary.tsx

```diff
@@ -399,7 +400,7 @@ function WantedSummary(props: WantedSummaryProps) {
       iconSize={14}
       style={{ fontSize: 14, height: 48, width: '100%', marginBottom: 20 }}
       endingIcon={'paid'}
-      text={paid ? 'Mark Unpaid' : 'Mark Paid'}
+      text={paid ? unpaidString : paidString}
```

Comment:
This does not fix this, make the change on https://github.com/stakwork/sphinx-tribes/blob/6073eb3a19fdd6db15f94b47a94767116e25d2a7/frontend/app/src/people/widgetViews/summaries/wantedSummaries/CodingBounty.tsx. In line 788 change isMarkPaidSaved to paid

Reply:
No actually it doesn't I checked on local the Mark as unpaid is not showing up ,I'm investigating it